modules/transport.js

```diff
@@ -535,7 +533,7 @@ Transport.prototype.shared = {
 	postBlock(query) {
 		query = query || {};
 		library.schema.validate(query, definitions.WSBlocksBroadcast, err => {
```

Comment:
Validation with `definitions.WSBlocksBroadcast` expects block as specified in `Block` definition, in P2P layer we are sending it in reduced form. The result is validation failing, with error: 
```
"err": [{
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["ip"],
								"message": "Missing required property: ip",
								"path": ["peer"]
							}, {
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["totalForged"],
								"message": "Missing required property: totalForged",
								"path": ["block"]
							}, {
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["reward"],
								"message": "Missing required property: reward",
								"path": ["block"]
							}, {
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["totalFee"],
								"message": "Missing required property: totalFee",
								"path": ["block"]
							}, {
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["totalAmount"],
								"message": "Missing required property: totalAmount",
								"path": ["block"]
							}, {
								"code": "OBJECT_MISSING_REQUIRED_PROPERTY",
								"params": ["numberOfTransactions"],
								"message": "Missing required property: numberOfTransactions",
								"path": ["block"]
							}],
							"module": "transport",
							"query": {
								"block": {
									"payloadHash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
									"timestamp": 56307470,
									"previousBlock": "11110474398246850590",
									"generatorPublicKey": "3be2eb47134d5158e5f7d52076b624b76744b3fba8aa50791b46ba21408524c9",
									"blockSignature": "061d3cf7b976f06b0f95b3b78027a5de95634f0c26539a285a38ece7742c3d1c34766d7dc9a557f00cb5aab4a90aae2d08b357b99876cad684e68ef2d1f58300",
									"height": 3,
									"id": "9408118491045224087",
									"relays": 1
								},
								"peer": {
									"os": "darwin16.7.0",
									"version": "0.0.1",
									"wsPort": 5001,
									"httpPort": 4001,
									"height": 2,
									"nethash": "198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d",
									"broadhash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
									"minVersion": ">=0.0.0",
									"nonce": "BjMSxFfaiSfuQhpD",
									"state": 2
								}
							}
						}
```
Schema needs to be changed to accept block in reduced form without fields:
- totalForged
- reward
- totalFee
- totalAmount
- numberOfTransactions